.travis.yml

```diff
@@ -1,7 +1,6 @@
 language: java
 jdk:
  - openjdk7
```

Comment:
don't you think that OpenJDK must be removed instead?

Reply:
Honestly, I don't have any strong opinion about that.  I personally have left JDK7 already, though.

Reply:
No worries, I'm finally back and now about to completely rework the travis.yml